keyring/keychain.go

```diff
@@ -2,27 +2,15 @@
 
 package keyring
 
-// https://developer.apple.com/library/mac/documentation/Security/Reference/keychainservices/index.html
-
-/*
-#cgo CFLAGS: -mmacosx-version-min=10.6 -D__MAC_OS_X_VERSION_MAX_ALLOWED=1060
-#cgo LDFLAGS: -framework CoreFoundation -framework Security
-
-#include <stdlib.h>
-#include <CoreFoundation/CoreFoundation.h>
-#include <Security/Security.h>
-*/
-import "C"
-
 import (
-	"errors"
 	"fmt"
 	"log"
-	"path/filepath"
-	"unicode/utf8"
-	"unsafe"
 
-	"github.com/mitchellh/go-homedir"
+	gokeychain "github.com/keybase/go-keychain"
+)
+
+const (
+	keychainAccessGroup = "ACE1234DEF.com.99designs.aws-vault"
```

Comment:
Yup, agreed. It's for sharing items between a family of apps, which we don't need.